components/atoms/volunteers.html

```diff
@@ -0,0 +1,81 @@
+<div class="component" title="volunteers-component" version="1.1">
+  <style>
+  
+   .volunteers img{
+    width: 100%;
+   }
+
+   .volunteers,.volunteers h1, .volunteers h2,.volunteers p{
+      margin:0;
+      padding-bottom:0.5rem;
+
+   }
+   
+   @media screen and (min-width: 500px) and (max-width: 699px){
+    .volunteers-wrapper{
+     display:grid;
+     grid-template-columns:1fr 1fr;
+     grid-gap: 15px;
+
+   }
+
+   }
+
+    @media screen and (min-width:700px) and (max-width: 1000px){
+    .volunteers-wrapper{
+     display:grid;
+     grid-template-columns:1fr 1fr 1fr;
+     grid-gap: 15px;
+
+   }}
```

Comment:
This looks very similar to the grid component that @24sint  made. Could you use a 3 column grid instead of writing new CSS?